### DISCUSSION TITLE
Discord integrated proximity chat plugin

### DISCUSSION THREAD

**Discussion by OSRS-Nick** [runelite/runelite#14959] [Ideas]
I want to talk with strangers in-game without the hassle of adding them and calling them on discord. 

A plugin that splits the map up into chunks such as "GE TRADING SQUARE" that means when you walk into the ge you would automatically be moved into the discord call for that area+world.

**Comment by damencs** [comment]
Where would you expect this type of plugin to be tied to? There are a LOT of map regions to add into something like this. It would be neat, but not something I think worthwhile, personally.

**Comment by marmiha** [comment]
Nice idea.

**Comment by juglia02** [comment]
don't support
